guides/service design at dxw

```diff
@@ -0,0 +1,42 @@
+
+## 1. **Prototype the service**
+- Before any prototyping begins, service designers may test the basis of the value proposition to ensure that the service works in theory. This check ensures it offers value to the end users/customers as well as the business.
+- Service designers **prototype and test service elements** that fall outside of a screen context, for example experiential prototypes of a physical space.
+- Service designers consider wider factors relevant to service success such as marketing, communications,  business process change and behavioural change. We work with relevant teams and specialists to design and prototype these elements of the service where possible.
+- A service designer may support the creation of screens and user flows, or would ideally support an interaction designer in doing this. We work to ensure that interaction designers, content designers and developers have clear user stories to work from, and that these are grounded in research insight. We also help define a prototyping strategy.
+
+## 1. **Tell the story and plan for implementation**
+- As we move from abstract to concrete, we craft artefacts that help others to **envision the proposed service**. We blend visual storytelling (e.g. storyboarding), which develops empathy for the people using and delivering the service, with technical diagrams (e.g. service blueprints), which serve as a detailed guide towards the implementation of the experience.
+- Service design helps to **build a narrative** of the service. Reflecting on the initial problem statement, a service designer can create a compelling presentation that explains how the problems identified at the start of the project have the potential to be resolved through the proposed service, explaining the rationale for the design decisions along the way.
+- Service designers consider factors that contribute to the service’s **future success**. We can create guides to support its implementation, prepare training that staff may need in order to deliver the service, and recommend organisational structures to ensure it will be properly managed in the future.
+- We also consider factors that could limit the success of the service and help teams mitigate them through **risk management**.
+- Services never exist in a vacuum, and their success depends on good alignment with the technology and processes that surround them. Service designers help manage these **technical and non-technical interfaces**.
+
+## 1. **Beta, live and beyond…**
+- Service designers like to be involved throughout the beta and live phases of a new service to help ensure that **performance is tracked and outcomes measured**.  We help service teams set up Key Performance Indicators and the data streams required to track them.
+- In a similar way we can also **support teams running existing services** to identify and implement improvements. 
```

Comment:
+ add this one - When those services require the transformation of an existing service the service designer plays an important role helping to define channel shift strategies for transitioning to a digital service.

Reply:
done